.github/workflows/build.yml

```
@@ -39,3 +40,7 @@ jobs:
         python tests/CI/test_fit.py
         python tests/CI/test_recurrent.py
         python tests/CI/test_critical.py
+    - name: Test CLI for errors
+      run: |
+        python setup.py install
```

Instead of installing, it's better to use `develop`.

```suggestion
        python setup.py develop
```